numpy/core/src/umath/override.c

```diff
@@ -45,9 +45,9 @@ normalize___call___args(PyUFuncObject *ufunc, PyObject *args,
      */
     int i;
     int not_all_none;
-    int nin = ufunc->nin;
-    int nout = ufunc->nout;
-    int nargs = PyTuple_GET_SIZE(args);
+    npy_intp nin = ufunc->nin;
+    npy_intp nout = ufunc->nout;
+    npy_intp nargs = PyTuple_GET_SIZE(args);
```

Comment:
Ok, I think I'm otherwise done here. I've given up on making `PyArray_LookupSpecial` behave more like `_PyObject_LookupSpecial`, because it messes with our `if (override == ndarray.__array_ufunc__)` check in ways that I can't debug